lib/offsite_payments/integrations/hi_trust.rb

```diff
@@ -31,7 +31,7 @@ def initialize(order, account, options = {})
           add_field('Type', 'Auth')
 
           # Capture the payment right away
-          add_field('depositflag', options[:deposit_flag] || '0')
+          add_field('depositflag', '1')
```

Comment:
Always be `1`, we didn't give them the option of passing in an option as setting manual or auto capture is not a setting supported for offsite gateways.